webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -635,6 +639,20 @@ Kommunicate.richMsgEventHandler = {
                 postBackData[name] = data[name];
             }
         }
+       if(index!=-1){
+        if (formElements[index].dataset.errorText) {
```

Comment:
if (index!=-1 && formElements[index].dataset.errorTex) {}

we can use this

Reply:
this is done @Sanjeevsinghnegi